external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -174,6 +177,8 @@ def build_model(
         config.input_variables,
     )
 
+    # note that n_features_out=1 is not used, as we connect subsequent layers
+    # to the last hidden output layer
```

Comment:
Thanks for adding a comment, suggesting here because it's documenting code I wrote.
```suggestion
    # note that n_features_out=1 is not used, as we want a separate output
    # layer for each variable (norm_output_layers)
```